src/bench/addrman.cpp

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2020-2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <addrman.h>
+#include <bench/bench.h>
+#include <random.h>
+#include <util/time.h>
+
+#include <vector>
+
+/* A "source" is a source address from which we have received a bunch of other addresses. */
+
+static constexpr size_t NUM_SOURCES = 64;
+static constexpr size_t NUM_ADDRESSES_PER_SOURCE = 256;
+
+static std::vector<CAddress> g_sources;
+static std::vector<std::vector<CAddress>> g_addresses;
+
+static void CreateAddresses()
+{
+    if (g_sources.size() > 0) { // already created
+        return;
+    }
+
+    FastRandomContext rng(uint256(std::vector<unsigned char>(32, 123)));
```

Comment:
nit: I think you can just set 
```suggestion
    FastRandomContext rng(/* deterministic */ true);
```

or is there a reason for the seed?

Reply:
Right, `/* deterministic */ true` would have been better. I will have this in mind and also change this code if I am touching it again.

There is no need for the chosen seed - it only has to be the same every time to reduce noise in performance tests.

Thanks!